open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java

```diff
@@ -18,168 +18,173 @@
-        HBox.setHgrow(header, Priority.ALWAYS);
-        return mainHeader;
-    }
-
-    /**
-     * creates custom styled text for the shortcut key menu.
-     * 
-     * @param buttonText the text to be set.
-     * @return styled HBox containing text or a styled button.
-     */
-    public HBox createCustomText(String buttonText)
-    {
-        HBox comboText = new HBox();
-        comboText.setMinHeight(30);
-        comboText.setSpacing(8);
-        comboText.setAlignment(Pos.CENTER_LEFT);
-        if (!buttonText.equals(""))
-        {
-            comboText.getChildren().add(new IconDispButton(buttonText));
-        }
-        return comboText;
-    }
-
-    /**
-     * creates a styled java ImageView to match other display sizes.
-     * 
-     * @param string the image to load.
-     * @return an Image display with a locked size and ratio.
-     */
-    public ImageView customImageView(String string)
-    {
-        ImageView customView = new ImageView(string);
-        customView.setFitHeight(30);
-        customView.setFitWidth(30);
-        customView.setPreserveRatio(true);
-        return customView;
-    }
-
-    /**
-     * creates a box to bound text for the legend of shortcuts.
-     * 
-     * @param displayText the text to be set.
-     * @return a VBox containing the text, right justified.
-     */
-    public VBox createLeftLabel(String displayText)
-    {
-        VBox labelBox = new VBox();
-        labelBox.getChildren().add(new Label(displayText));
-        labelBox.setAlignment(Pos.CENTER_RIGHT);
-        return labelBox;
-    }
-
-    /**
-     * A custom button for putting text or images into buttons.
-     */
-    private class IconDispButton extends Button
-    {
-        private IconDispButton(String key)
-        {
-            setDisable(true);
-            setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
-            setTextFill(Color.web("white"));
-            setText(key);
-        }
-    }
+public class ControlTab {
+	/**
+	 * The JavaFX gridpane containing the shortcut key keys and keys in a column
+	 * arranged fashion. Since this pane is a child of another pane it must be
+	 * accessed via the get method when adding items to it outside of this class.
+	 */
+	private GridPane myDisplayArea;
+
+	/**
+	 * creates a custom styled button with special text,styling, and sizing.
+	 * 
+	 * @param text the icon to be set.
+	 * @return a custom styled Button with various FontIconEnums inside.
+	 */
+	public Button createIconButton(FontIconEnum text) {
+		Button customIcon = new Button();
+
+		final Label label = new Label(text.getFontCode());
+		label.setTextFill(Color.WHITE);
+		label.setOpacity(75);
+
+		final String fontName = "icons-" + text.getFont().getFontName().replaceAll("\\s", "-");
+		label.getStyleClass().addAll(fontName);
+		label.setStyle("-fx-font-size: 16 px;");
+
+		customIcon.setGraphic(label);
+		customIcon.setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
```

Comment:
This should be a constant string, its used in 3 places in the file

Reply:
replaced with fxTheme